image_width: 1920
image_height: 1080
camera_name: gopro
camera_matrix:
  rows: 3
  cols: 3
  data: [857.48296979 ,  0.0,         968.06224829,  0.0,        876.71824265, 556.37145899,0,0,1]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-2.57614020e-01,  8.77086999e-02, -2.56970803e-04, -5.93390389e-04, -1.52194091e-02]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.000000, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000, 0.000000, 0.000000, 1.000000]
projection_matrix:
  rows: 3
  cols: 4
  data: [857.48296979 ,  0.0,         968.06224829,  0.0,0.0,        876.71824265, 556.37145899,0,0,0,1,0]

